### ISSUE TITLE
How to implement Method like TokenClassification

### ISSUE THREAD

**Issue by Irvingwangjr** [thinking-machines-lab/tinker-feedback#18]
I'm trying to use tinker api to implement a VLA model, which will project the logits to a action classifier. But tinker only return logProbes. I'm trying to seek some suggestion from here
